crates/core/tedge_actors/src/servers/message_boxes.rs

```diff
@@ -281,7 +286,12 @@ mod tests {
             }
 
             fn client_box(&mut self) -> SimpleMessageBox<i32, i32> {
-                self.box_builder.new_client_box()
+                let mut test_box = SimpleMessageBoxBuilder::new("Test box", 16);
+                test_box.connect_sink(
+                    NoConfig,
+                    &self.box_builder.connect_client(test_box.get_sender()),
+                );
+                test_box.build()
```

Comment:
Same comment as above.

Reply:
This is a residue of my attempt to fully remove the `new_client_box()` test helper.

Reverted.